app/models/user.rb

```diff
@@ -2,15 +2,27 @@
 #
 # Table name: users
 #
-#  id         :integer          not null, primary key
-#  uid        :string           not null
-#  email      :string           not null
-#  admin      :boolean          default(FALSE)
-#  created_at :datetime         not null
-#  updated_at :datetime         not null
+#  id                 :integer          not null, primary key
+#  uid                :string           not null
+#  email              :string           not null
+#  admin              :boolean          default(FALSE)
+#  created_at         :datetime         not null
+#  updated_at         :datetime         not null
+#  sign_in_count      :integer          default(0), not null
```

Comment:
It comes along free with Devise. I thought it might be helpful in debugging user problems. If this app had any privacy sensitivity associated I might lean towards not tracking, but since it's "upload your thesis" I think it's legit to have this info for debug purposes. I'm happy to discuss concerns with it though. It is indeed _not_ in the spec ;)